### PR TITLE
Add workload and service dashboard

### PR DESCRIPTION
**Pull request by douglas-reid**
This PR refactors the existing grafana dashboards in the following manner:
- Adds a "service" dashboard that focuses on displaying info relevant to service consumers.
- Adds a "workload" dashboard that focuses on incoming workloads and outgoing service calls.
- Sets-up the "mesh" dashboard to consist of tables that link to individual dashboards.

The new dashboards themselves are not segmented into separate dashboards for TCP and HTTP/GRPC any more. As a TCP service may make HTTP calls out, and vice-versa, that distinction would lead to a lot of duplication and (even more) complexity. In the future, we can use the bandwidth placeholders to report TCP stats for HTTP services as well.

This PR also updates the newly committed "conditional" in favor of "conditionalString".

There are several known issues exposed by this PR:
- `destination.service.host` is being set to `unknown` for all TCP calls (client and server).
- `connection.mtls` is set to false for `reporter="client"` calls. This will need documentation.

Screenshots:

mesh-dash-top
mesh-bottom-dash
service-dash-top
service-dash-bottom
tcp-client-workload-dash
tcp-workload-dash
workload-middle-dash
workload-bottom-dash

### CODE REVIEW

File: install/kubernetes/helm/istio/charts/mixer/templates/config.yaml [hunk 1]
```diff
@@ -558,7 +558,7 @@ spec:
     - destination_workload_namespace
     - destination_version
     - destination_service
-    - destination_service_name
+    - destination_service_nam e
```
Comment (gargnupur) [inline install/kubernetes/helm/istio/charts/mixer/templates/config.yaml]: destination_service_nam e ? left by mistake?

Comment (douglas-reid) [reply install/kubernetes/helm/istio/charts/mixer/templates/config.yaml]: yikes. thanks.

**Comment by kyessenov** [comment]
@rshriram we're observing that plugins do not receive service in the plugins for TCP listeners. Any idea?

**Comment by hklai** [comment]
Merging this manually. We are creating 1.0 branch at this SHA.

**Pull request merged**